tests/abi/enc/enc.cpp

```diff
@@ -9,6 +9,17 @@
 #include "../abi_utils.h"
 #include "abi_t.h"
 
+OE_EXTERNC_BEGIN
+// By default, assume avx is available. This is the case with most processors
+// with SGX support.
+bool oe_is_avx_enabled = true;
```

Comment:
Don't you need the similar check as #3772? If `oe_is_avx_enabled` is always set to `true`, then the test could still fail on the machine without AVX support.

Reply:
Never mind. On the host, `oe_is_avx_enabled` is global and set by the check in #3772. I'm just confused by that both the enclave and the host use the same variable name.

Reply:
`oe_is_avx_enabled` defaults to true. It is set to the correct value by the host via `enclave_set_oe_is_avx_enabled`